src/mscorlib/shared/System/BitConverter.cs

```diff
@@ -267,7 +269,7 @@ public static int ToInt32(ReadOnlySpan<byte> value)
         {
             if (value.Length < sizeof(int))
                 ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.value);
-            return Unsafe.ReadUnaligned<int>(ref value.DangerousGetPinnableReference());
+            return Unsafe.ReadUnaligned<int>(ref MemoryMarshal.GetReference(value));
```

Comment:
Does this compile? I would expect that you will get readonly vs. non-readonly mismatch.

Reply:
Yes. Need to add:
```C#
ref T AsRef<T>(in T source)
```

Doing that now.